._fake/config.fsx

```diff
@@ -0,0 +1,19 @@
+namespace FSharpVersionUtils.Fake.Config
+
+#r @"./packages/FAKE/tools/FakeLib.dll"
+
+open Fake
+open System.IO
+
+module Common =
+  let RootDir = Directory.GetCurrentDirectory()
```

Comment:
Potentially nitpicky, but would this fit in 80 columns if we brought it down to one line?